### ISSUE TITLE
Discussion Group: first steps towards more participatory democracy here in NYC

### ISSUE THREAD

**Issue by ebarry** [ProgressiveHackNight/breakout-groups#16]
### About the group
Let's discuss the very first, simple steps along the way to the exciting participatory democratic wins happening in Taiwan and Spain (Madrid). 

### Group leader/s
Well, there are many people already discussing these issues. In early June [at PDF here in NYC, Audrey Tang gave a main stage presentation](https://www.youtube.com/watch?v=5DkhUO7LiGs&list=PLcvJwS9IjFNZmDSxd37-InpYubosxteci&index=31), and i joined her and TTCat and @colinmegill for a panel that got many more people excited. I've invited these folks to the July 18 ProgHacknight.
I'm @ebarry (http://twitter.com/lizbarry).

### Who we're looking for
People interested in facilitation would be appreciated!

### Tools

At this point we are interested in first steps. Let's compare our notes on in-person facilitation methods. Let's geek out on how in-person facilitation methods are being used in Taiwan in a mixed reality context. 

### Relevant Links
https://civichall.org/civicist/vtaiwan-democracy-frontier/
https://www.opendemocracy.net/democraciaabierta/bernardo-guti-rrez/madrid-as-democracy-lab

### Where we meet
TBD

**Comment by patcon** [comment]
Woo! Made a thing for ongoing documentation of whatever this turns into: http://beta.hackfoldr.org/nyc-participatory-democracy

Any thoughts on whether this fits with how we imagine tonight going?

**Comment by noneck** [comment]
Have you seen http://nycroadmap.us?

Four years ago, we forged a citywide conversation to discuss the future of tech in government and community. Currently, BetaNYC is making plans to revise this roadmap by hosting a 5 borough tour...

**Comment by ebarry** [comment]
hey folks! a few of us including @patcon and @DZNarayanan are going to come by tomorrow evening to pick up on this topic, looking forward to seeing people! cc @noneck @devinbalkind @misscs

**Comment by noneck** [comment]
wow! where are you guys discussing these issues?

less than 24 hours notice is less than ideal. 

—

**Comment by ebarry** [comment]
HI Noel!!! Tomorrow eve we're joining ProgHacknight at 99 Madison, 15th floor. 
Otherwise for communications we use the g0v slack: for local issues the #vnyc channel, and for people working in other non-taiwan locations the #vnetwork channel. 
We are also falling into a Friday daytime routine at New Lab followed by walking back to cook dinner at my house

**Comment by patcon** [comment]
Also, in case it's simpler to stay on top: We're sharing calendars here: https://trello.com/c/Y0RtqdnV/7-calendar

Note that there is a full calendar to browse, plus separate ical feeds for gcal, depending on which types of dates you're interested in

fwiw, we just had a realization that we need to be better with notifications:
https://g0v-tw.slack.com/archives/C8FBZG4NS/p1515806615000128

Would the ical feed work for you, or is there a better way for meetings to get on your radar?

**Comment by noneck** [comment]
it is a shame that these efforts aren’t better integrated with activities around the People’s roadmap. 

—

**Comment by patcon** [comment]
Yeah, I hear ya. For context, many of the efforts in that calendar are under a banner of vNetwork, not vNYC -- the intention was to have places for high-level convo not exclusive to a specific place, so that it's still relevant to people like myself organizing in Toronto.
